src/main/java/no/unit/nva/doi/CrossRefClient.java

```diff
@@ -39,15 +39,24 @@ public CrossRefClient(HttpClient httpClient) {
         this.httpClient = httpClient;
     }
 
-    public Optional<String> fetchDataForDoi(String doiIdentifier) throws URISyntaxException {
+    /**
+     * The method returns the object containing the metadata (title,author etc) of the publication with the specific
```

Comment:
Insert space and trailing comma and abbreviation full stop: title, author, etc.

Reply:
fixed.